health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/DeliveryDetailsSummary.js

```
@@ -85,45 +86,93 @@ function reverseDeliveryRemap(data, t) {
+    const deliveries = cycle.deliveries?.map((delivery, deliveryIndex) => {
+        const doseCriteria = delivery.doseCriteria?.flatMap((criteria, ruleKey) => {
+            const products = criteria.ProductVariants.map((variant, key) => ({
+                key: key + 1,
+                count: 1,
+                value: variant.productVariantId,
+                name: variant.name
+            }));
+
+            const condition = criteria.condition;
+            let conditionParts = condition.split("and").map(part => part.trim()); // Split by 'and' and trim spaces
+            let rules = [];
```

Biome

[error] This let declares a variable that is only assigned once.

'rules' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)